doc/source/methods/index.rst

```diff
@@ -40,11 +40,27 @@ Geometry shapes functions are used for
 defining and modifying Adaptive Templates geometries in Python.
 For descriptions of the functions, see :ref:`ref_geometry_shapes`.
 
+Utility functions
+------------------
+
+A number of utility functions are available to the user.
+For descriptions, see
```

Comment:
```suggestion
A number of utility functions are available.
For more information, see
```
Avoid "user" but use "you" instead--unless you are distinguishing between user types  (administrators versus users).